shell/browser/extensions/api/runtime/atom_runtime_api_delegate.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Is it okay to leave these copyright headers introduced from the extensions shell code? Or should they be replaced with the GitHub copyright?

Reply:
🤷‍♂